pom.xml

```diff
@@ -24,9 +24,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.nirima.docker-java</groupId>
+            <groupId>com.github.docker-java</groupId>
             <artifactId>docker-java</artifactId>
-            <version>0.10.5-SNAPSHOT</version>
+            <version>0.10.5</version>
```

Comment:
`0.10.5` version is released... if we want another one, we may use `0.10.6-SNAPSHOT`